setup.cfg

```diff
@@ -53,7 +53,12 @@ source-dir = docs
 build-dir = docs/_build
 
 [tool:pytest]
-addopts = --cov=django_select2 --cov-report xml
+addopts =
+    tests
+    --doctest-glob='*.rst'
+    --doctest-modules
```

Comment:
I don't see any doctests added in this pull request. What's the idea behind this change?

Reply:
True, I just added it, in case I write some. I might write some later :)